src/modules/admin/components/UserListItem/UserListItem.tsx

```diff
@@ -79,8 +91,35 @@ const UserListItem = ({
 
   const { profile: { username = undefined, displayName = undefined } = {} } =
     user || {};
+
+  const handleClick = useCallback(() => {
+    if (typeof callbackFn === 'function') {
+      callbackFn(address);
+    }
+  }, [address, callbackFn]);
+
+  const handleKeyPress = useCallback(
+    (evt: KeyboardEvent<HTMLElement>) => {
+      if (evt.key === ENTER) {
+        callbackFn(address);
+      }
+    },
+    [address, callbackFn],
+  );
+
+  const rowProps = callbackFn
+    ? {
+        onClick: handleClick,
+        onKeyPress: handleKeyPress,
+        tabIndex: 0,
+      }
+    : {};
+
   return (
-    <TableRow className={styles.main}>
+    <TableRow
+      className={getMainClasses({}, styles, { hasCallbackFn: !!callbackFn })}
+      {...rowProps}
+    >
       <TableCell className={styles.userAvatar}>
         <UserAvatar size="xs" address={address} user={user} />
```

Comment:
According to the design in Figma `size="xs"`, but maybe we don't wanna change that since it should be a global decision. Up to you!